### PR TITLE
Upload Migration Alignments

### PR DESCRIPTION
**Pull request by Claudia-WE**

### CODE REVIEW

**Review by JohannaOtt — CHANGES_REQUESTED** [review]
Changes to be done are documented in relevant tickets SVC-1230, SVC-1231 and SVC-1232

**Comment by JohannaOtt** [comment]
@Claudia-WE I can see that some changes were directly pushed to the main branch and not added to this PR - for example in this folder https://github.com/wetransform-os/xplanung-transformationswerkzeuge/tree/main/Versionsmigration/5.0-5.4
Not sure if and how that can be properly reverted so that changes are added via a PR. Maybe you can have a look at this together with Florian.

File: Versionsmigration/5.2-5.4/xplangml-migration-52-54.halex [hunk 1]
```diff
@@ -24,6 +24,21 @@
         <setting name="source">https://repository.gdi-de.org/schemas/de.xleitstelle.xplanung/5.4/XPlanung-Operationen.xsd</setting>
         <setting name="contentType">eu.esdihumboldt.hale.io.xsd</setting>
     </resource>
+    <resource action-id="eu.esdihumboldt.hale.io.instance.read.source" provider-id="eu.esdihumboldt.hale.io.gml.reader">
```
Comment (JohannaOtt) [inline Versionsmigration/5.2-5.4/xplangml-migration-52-54.halex]: I would vote for not committing source data loaded.

File: Versionsmigration/5.0-5.4/xplangml-migration-50-54.halex [hunk 2]
```diff
@@ -0,0 +1,1509 @@
...
+        <setting name="contentType">eu.esdihumboldt.hale.io.xsd</setting>
+    </resource>
+    <resource action-id="eu.esdihumboldt.hale.io.schema.read.target" provider-id="eu.esdihumboldt.hale.io.xsd.reader">
+        <setting name="charset">UTF-8</setting>
+        <setting name="onlyElementsMappable">true</setting>
+        <setting name="resourceId">b125c25e-d851-463a-b30a-a8f24e7d5f1d</setting>
+        <setting name="source">https://repository.gdi-de.org/schemas/de.xleitstelle.xplanung/5.4/XPlanung-Operationen.xsd</setting>
+        <setting name="contentType">eu.esdihumboldt.hale.io.xsd</setting>
+    </resource>
+    <resource action-id="eu.esdihumboldt.hale.io.instance.read.source" provider-id="eu.esdihumboldt.hale.io.gml.reader">
```
Comment (JohannaOtt) [inline Versionsmigration/5.0-5.4/xplangml-migration-50-54.halex]: Again, I would avoid to load data.

**Review by JohannaOtt — CHANGES_REQUESTED** [review]

**Comment by Claudia-WE** [comment]
Closing, created seperate pull requests for each alignment

**Pull request closed without merging**